lib/cocoapods/installer/xcode/pods_project_generator/file_references_installer.rb

```diff
@@ -308,3 +312,28 @@ def common_path(paths)
     end
   end
 end
+
+# If we have an non-empty .docc folder, remove all paths under the folder
+# but keep the folder itself
+def merge_to_docc_folder(paths)
```

Comment:
why does this need to be a separate method and not included with the logic above?